storage/src/test/java/tech/pegasys/teku/storage/store/AbstractStoreTest.java

```diff
@@ -42,7 +43,7 @@
 import tech.pegasys.teku.storage.api.StubStorageUpdateChannel;
 
 public abstract class AbstractStoreTest {
-  protected final Spec spec = TestSpecFactory.createMinimalPhase0();
+  protected final Spec spec = TestSpecFactory.createMinimalEip4844();
```

Comment:
Shouldn't we keep at at `Phase0` and override it if required?

Reply:
It was much simpler this way.. since this is an abstract test class and we are dealing with `protected final` fields.